cetmix_tower_server/models/cx_tower_plan_line.py

```diff
@@ -46,6 +47,34 @@ class CxTowerPlanLine(models.Model):
         "will be launched. e.g.: {{ odoo_version}} == '14.0'",
     )
 
+    @api.constrains("command_id")
+    def _check_command_id(self):
+        """
+        Check recursive plan line execution. If the command refers to another plan,
+        make sure there are no recursive references.
+        """
+        for line in self:
+            visited_plans = set()
+            self._check_recursive_plan(line.command_id, visited_plans)
+
+    def _check_recursive_plan(self, command, visited_plans):
```

Comment:
Comment not an issue: provably we need to improve test coverage here and create a separate test for this function with several cases. What do you think?